setup.py

```diff
@@ -1398,6 +1461,29 @@ def main():
         "packaged/autograd/*",
         "packaged/autograd/templates/*",
     ]
+
+    if BUILD_LIBTORCH_WHL:
+        modified_packages = []
+        for package in packages:
+            parts = package.split(".")
+            if parts[0] == "torch":
+                modified_packages.append("libtorch" + package[len("torch") :])
```

Comment:
Here in place of "libtorch" perhaps would be better to use ``DEFAULT_PACKAGE_NAME``